### DISCUSSION TITLE
在clash系上游删库的当下，ppgw会换其他方案吗？

### DISCUSSION THREAD

**Discussion by YZY1360** [kkkgo/PaoPaoGateWay#33] [想法]
如题
大佬提供的全套方案对我很有帮助，想问一下这个项目的未来计划。

**Comment by kkkgo** [comment]
clash meta似乎并没有删库，wiki只是整活变成原神了 https://wiki.metacubex.one/   
暂时先不动了，如果实在都删了，有可能换sing-box

**Reply by seankuo1** [reply]
变成只读了，希望只是避风头

**Reply by hsj1992** [reply]
sing-box我之前也看过，发现在各个操作系统上的GUI客户端使用都挺一言难尽的...
nekoray的作者对sing-box的评价也挺辛辣。

**Comment by Lovest20018** [comment]
meta用来当内核的感觉太重了吧？sing-box倒是可以考虑-大佬要么来个测试版试下？

**Comment by kkkgo** [comment]
clash meta 已经恢复更新，只是换了个名字。
仓库地址 https://github.com/MetaCubeX/mihomo/tree/Alpha
releases下载 https://github.com/MetaCubeX/mihomo/releases

**Comment by Lovest20018** [comment]
大佬考虑默认核心换mihomo了吗？

**Reply by kkkgo** [reply]
你可以自行下载meta内核放入待生成的目录。出于稳定性和体积的考虑，docker镜像目前依然保持原版内核。
